model/types/group_test.go

```diff
@@ -26,26 +27,26 @@ func ExampleVolume_Format() {
 	// /brace/yourselves:/winter/is/coming:rw
 }
 
-func TestMembers_RemoveDuplicates(t *testing.T) {
-	// Given
-	givenMembers := Members{
-		{User: "batman", Role: MemberModeratorRole},
-		{User: "batman", Role: MemberUserRole},
-		{User: "batman", Role: MemberModeratorRole},
-		{User: "superman", Role: MemberUserRole},
-		{User: "superman", Role: MemberUserRole},
-		{User: "robin", Role: MemberModeratorRole},
-	}
-
-	// When removing duplicates
-	actualMembers := removeDuplicatesMember(givenMembers)
-
-	// Then duplicates should haven been deleted
-	expectedMembers := Members{
-		{User: "batman", Role: MemberModeratorRole},
-		{User: "superman", Role: MemberUserRole},
-		{User: "robin", Role: MemberModeratorRole},
-	}
-	assert.Equal(t, actualMembers, expectedMembers, "Duplicates should have been deleted")
-
-}
+var _ = Describe("Group", func() {
+	Context("Given a list of members", func() {
+		givenMembers := types.Members{
+			{User: "batman", Role: types.MemberModeratorRole},
+			{User: "batman", Role: types.MemberUserRole},
+			{User: "batman", Role: types.MemberModeratorRole},
+			{User: "superman", Role: types.MemberUserRole},
+			{User: "superman", Role: types.MemberUserRole},
+			{User: "robin", Role: types.MemberModeratorRole},
+		}
+		Context("When I remove duplicates", func() {
+			actualMembers := types.RemoveDuplicatesMember(givenMembers)
+			It("Then duplicates should haven been deleted", func() {
+				expectedMembers := types.Members{
+					{User: "batman", Role: types.MemberModeratorRole},
+					{User: "superman", Role: types.MemberUserRole},
+					{User: "robin", Role: types.MemberModeratorRole},
+				}
+				Ω(actualMembers).Should(Equal(expectedMembers))
```

Comment:
ctrl+shit+u+03A9 sous linux, je vois pas du tout le problème 😸 
Je vais le faire mais c'est quand même con l’écriture Ω().should est plus fluent, je comprend pas pourquoi ils ont fait ça , surement pour le trip.

@matcornic  m'a dit que vous allez principalement faire du back, du coup je vais ajouter la couverture des tests dans le makefile et ensuite je vais rationnaliser le front: 
- react router v4 à mettre
- refacto des reducers pour utiliser le combineReducers
- repasser sur les composant pour ne faire les fetchs uniquement dans des composant "Containers" (sans code metier)
- observables à mettre à la place des thunks (plus sympas si ya des websockets)
- passage à typescript
- mise en place de jest pour les tests unitaire